tests/validation/tests/v3_api/test_bkp_restore.py

```diff
@@ -65,7 +81,85 @@ def test_bkp_restore():
     assert len(workload_list) == 0
 
 
-@pytest.fixture(scope='module', autouse="True")
+def test_bkp_restore_s3_with_creds(create_project_client_and_cluster_s3_with_creds):
+    validate_s3_create_restore_delete()
+
+
+def test_bkp_restore_s3_with_iam(create_project_client_and_cluster_s3_with_iam):
+    validate_s3_create_restore_delete()
+
+
+def validate_s3_create_restore_delete():
```

Comment:
Can the S3 checks in this method be folded into the existing validate_test_bkp_restore by making it take a mode parameter that can be set to local Vs s3?